client/src/components/OpportunitiesCard.js

```diff
@@ -1,15 +1,13 @@
 import React from "react";
```

Comment:
change this file name too .. it's `OpportunityCard` not `opportunities` - the component is only displaying one

Reply:
Done